raiden-cli/src/index.ts

```diff
@@ -322,10 +329,14 @@ function createRaidenConfig(
       },
     };
 
-  let mediationFees: { [token: string]: { flat: BigNumberish } } = {};
+  type Feetype = 'flat' | 'proportional';
+  let mediationFees: { [token: string]: { [K in Feetype]?: BigNumberish } } = {};
   for (const [addr, flat] of Object.entries(argv.flatFee ?? {})) {
     mediationFees = { ...mediationFees, [addr]: { ...mediationFees[addr], flat } };
   }
+  for (const [addr, proportional] of Object.entries(argv.proportionalFee ?? {})) {
```

Comment:
@andrevmatos I guess it's too late for changing this, but what do you think about having a parent configuration options called `mediationFees` and then have `flatFee` renamed to `flat` as child as well as `proportionalFee` renamed to `proportional`? So because atm this is like 99% the same code except the (re)naming, but it will be required to replicate this code for every new fee model we add. If we could align the names better it would be a single code block once and it works for any new model. 
Would be cool. But I understand if you don't want to change this now.

Reply:
Okay, I understand. So we can blame them. 😛